pkg/rancher/migrations.go

```diff
@@ -39,15 +45,33 @@ func forceUpgradeLogout(configMapController controllerv1.ConfigMapController, to
 		}
 	}
 
-	// we only want to migrate if the previous installation was earlier than the migration version
+	// parse out the start and end time
+	lastMigrationStart, _ := strconv.ParseInt(cm.Data[migrationStartedKey], 10, 64)
```

Comment:
Use the helpers for all the conversions https://pkg.go.dev/strconv#hdr-Numeric_Conversions
`Atoi`
`Itoa`
It keeps it consistent and it's much faster than using sprintf which relies on reflection.